src/libs/utils.ts

```diff
@@ -1,8 +1,11 @@
+import * as core from '@actions/core';
 export function invariant(
   condition: any,
   message?: string,
 ): asserts condition {
   if (!condition) {
-    throw new Error(message);
+    core.setFailed(message);
+    process.exit();
+    // throw new Error();
```

Comment:
Don't know if this is the way we'd like to do it, or if we want to throw and catch it somewhere else?

Reply:
I think this is much more in keeping with how actions should be written - right?

Reply:
Yes, but considering this, we could do it the other way too.

https://github.com/pulumi/actions/blob/a7bbd3813555fb55dd615ed5b07c3f6e3cde0b90/src/main.ts#L50-L58

I don't know. We can probably come back to this.